tests/unit_tests/test_wave_06.py

```diff
@@ -76,7 +76,16 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert jesse.inventory[-1] == item_c
+    assert jesse.inventory[0] == item_d
+    assert jesse.inventory[1] == item_e
+    assert tai.inventory[1] == item_b
+    assert tai.inventory[0] == item_a
+    assert tai.inventory[-1] == item_f
```

Comment:
Rather than asserting on specific places in the inventory list, it would be better to do `assert item_c in jesse.inventory`. We can't know for sure how items are added to the inventory and it could change under the hood.